pkg/utils/validation/cidr/disjoint.go

```diff
@@ -20,6 +20,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
+const defaultVpnRange = "192.168.123.0/24"
```

Comment:
Maybe this constant should be moved to the `pkg/apis/core/v1beta1/constants` package

Reply:
Should it then be reused in both files?

Reply:
Yep :)

Reply:
Moved constant to constants package as requested.